### ISSUE TITLE
EnvGen + doneAction:1 + gate==0 causes spurious /n_off notification

### ISSUE THREAD

**Issue by jamshark70** [supercollider/supercollider#334]
These conditions together cause an /n_off notification to be sent from the server at the beginning of the node. But the node is still running -- it is not paused.
1. EnvGen with a gate argument and doneAction == 1.
2. The synth is started with gate == 0.

This n_off should be omitted.

Reproducible in both scsynth and supernova.

Reproducer:

```
SynthDef(\pausefail, { |gate = 0|
    var freq_eg = EnvGen.kr(
        Env([200, 400, 300], [2, 2], \exp, releaseNode: 1),
        gate,
        doneAction: 1
    );
    Poll.kr(2, freq_eg);
}).add;

f = { |msg|
    if(#['/n_go', '/n_end', '/n_on', '/n_off'].includes(msg[0])) {
        msg.postln;
    };
};

thisProcess.addOSCRecvFunc(f);

// we get /n_go here -- OK
// and then, immediately /n_off

// THE /n_off IS WRONG -- it keeps posting Poll output
// so the node is definitely not paused

a = Synth(\pausefail, [gate: 0]);

a.run(false);  // /n_off, correctly, and Poll stops
a.run(true);   // /n_on, correctly, and Poll resumes

a.set(\gate, 1);  // polled value increases

a.set(\gate, 0);  // polled value decreases and node pauses (OK)

a.free;

// when done tracing
```

**Comment by telephon** [comment]
@jamshark70 can you test if the pull request fixes it for you?

**Comment by LFSaw** [comment]
Do I understand 0f69855 correctly that this issue is not fixed anymore?

**Comment by telephon** [comment]
Yes, unfortunately.

**Comment by telephon** [comment]
this is related: #2189.

**Comment by brianlheim** [comment]
closed in #4436, thanks @sonoro1234 !
